applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_potential_flow_utilities.cpp

```diff
@@ -84,6 +133,31 @@ KRATOS_TEST_CASE_IN_SUITE(ComputeLocalSpeedOfSound, CompressiblePotentialApplica
     KRATOS_CHECK_NEAR(local_speed_of_sound, 333.801138, 1e-6);
 }
 
+// Checks ComputeLocalSpeedofSoundSquared in utilities, local velocity that should be clamped
+KRATOS_TEST_CASE_IN_SUITE(ComputeLocalSpeedofSoundSquared, CompressiblePotentialApplicationFastSuite) {
+    Model this_model;
+    ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+    AssignFreeStreamValues(model_part);
+
+    double local_mach_number = 3.0;
```

Comment:
shouldnt this be square root of 3? You can do:

```suggestion
    const double local_mach_number_squared = 3.0;
```

and then you dont need to square it afterwards. You can check how you do it in the next test: ComputeLocalMachNumberSquared, i think there it is right